fenix/app/src/main/java/org/mozilla/fenix/home/topsites/TopSites.kt

```diff
@@ -374,9 +371,10 @@ private fun TopSiteFaviconCard(
                 shape = RoundedCornerShape(4.dp),
             ) {
                 if (topSite is TopSite.Provided) {
-                    FaviconBitmap(topSite)
+                    submitTopSitesImpressionPing(topSite, position)
```

Comment:
we were missing `submitTopSitesImpressionPing` from the original non Compose implementation

Reply:
This is already done in https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/home/topsites/TopSites.kt#343. This is done in an effect so that it wouldn't break previews